terraform-provider-conjur.rb

```diff
@@ -5,22 +5,31 @@
 class TerraformProviderConjur < Formula
   desc "Terraform provider for CyberArk Conjur"
   homepage "https://github.com/cyberark/terraform-provider-conjur"
-  version "0.5.0"
+  version "0.6.0"
   bottle :unneeded
 
-  if OS.mac? && Hardware::CPU.intel?
-    url "https://github.com/cyberark/terraform-provider-conjur/releases/download/v0.5.0/terraform-provider-conjur-0.5.0-darwin-amd64.tar.gz"
-    sha256 "13500229d45fdc5c2b3e9fe56c5ef263756f71cda35ff29e0fc2ec2519465674"
+  on_macos do
+    if Hardware::CPU.intel?
+      url "https://github.com/cyberark/terraform-provider-conjur/releases/download/v0.6.0/terraform-provider-conjur-0.6.0-darwin-amd64.tar.gz"
```

Comment:
I don't see these images on https://github.com/cyberark/terraform-provider-conjur/releases. Are they missing or is it me.

Reply:
Okay @rpothier I've created the release!